src/sys/windows/mod.rs

```diff
@@ -35,7 +35,9 @@ macro_rules! try_io {
         }
         result
     }};
-    ($self: ident.$method: ident ( $($arg: expr),* $(,)* ), $reregister_check: expr)  => {{
+   /* // This try_io variant does reregister if a certain condition is met
```

Comment:
You can just remove this, git history hold this code anyway.

Reply:
ok, removed.